styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -0,0 +1,38 @@
+{% block skipNav %}
+  {% include "@atoms/skip-nav.twig" %}
+{% endblock %}
+
+<div class="container">
+  {% block ribbon %}
+    {% include '@base/placeholder.twig' with {'placeholder':{'text':'Topic page  ribbon'}} %}
+  {% endblock %}
+
+  {% block header %}
+    {% include '@base/placeholder.twig' with {'placeholder':{'text':'Topic page header'}} %}
+  {% endblock %}
+
+  <main id="main-content" class="three-column" tabindex="-1" role="main">
```

Comment:
Please update class `.three-column` to be BEM-friendly and follow the other guidelines. `.ama__layout--three-column`?

Reply:
done